### PR TITLE
replace deprecated functions after TF SDK v2 upgrade #1

### PR DESCRIPTION
**Pull request by ivan-savciuc**

### CODE REVIEW

**Review by encima — APPROVED** [review]
If you can check over @ivan-savciuc and I will merge once you approve. We can ignore the hanging Travis job I think :P

**Review by ivan-savciuc — COMMENTED** [review]
Code after combining LGTM, do not know what is happening with TravisCI, we deleted the related config file and turned it off for this project.

**Pull request merged**
